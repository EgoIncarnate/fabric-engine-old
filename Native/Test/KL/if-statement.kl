function entry()
{
  var Integer foo;
  if ( 2 )
    foo = 17;
  else
    foo = 24;
  report foo;
}
