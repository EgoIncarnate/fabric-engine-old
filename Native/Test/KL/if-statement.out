GlobalList
  Function( 'entry' ('entry'), void )
    ParamList
    CompoundStatement
      StatementList
        StatementList
          VarDecl( 'foo', Integer )
        ConditionalStatement
          ConstSize( 2 )
          ExprStatement
            AssignOp(ASSIGN_OP)
              Var('foo')
              ConstSize( 17 )
          ExprStatement
            AssignOp(ASSIGN_OP)
              Var('foo')
              ConstSize( 24 )
        Report
          Var('foo')
17
